File: 백준/Silver/11650. 좌표 정렬하기/좌표 정렬하기.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').toString().trim().split('\n');
const nums = input.slice(1).map(x => x.split(' ').map(Number));
const result = nums.sort((a, b) => a[0] === b[0] ? a[1] - b[1] : a[0] - b[0]);
console.log(result.map(x => x.join(' ')).join('\n'));

File: 백준/Silver/11650. 좌표 정렬하기/좌표 정렬하기.py
import sys

N = int(input())
coords = [list(map(int, sys.stdin.readline().split())) for _ in range(N)]
sorted_coords = sorted(coords, key=lambda x: (x[0], x[1]))

print('\n'.join(map(lambda x: ' '.join(map(str, x)), sorted_coords)))